context.go

```diff
@@ -380,13 +404,16 @@ func (c *Context) Sqrt(d, x *Decimal) (Condition, error) {
 		ed.Add(approx, approx, New(819, -4))
 	}
 
+	// TODO(mjibson): Now we repeatedly improve approx. Our precision improves
```

Comment:
OK, that's a stylistic choice that I can live with, although I would do it differently.